templates/default/less/variables.less

```diff
@@ -340,3 +340,16 @@
 //##
 @il-counter-novelty-bg: @brand-warning;
 @il-counter-novelty-margin-top: -5px;
+
+//== Popover
+//
+//##
+@il-popover-min-width: 50px;
+@il-popover-min-height: 32px;
+@il-popover-max-width: 400px;
+@il-popover-background-color: @il-primary-container-bg;
+@il-popover-title-background-color: @il-popover-background-color;
+@il-popover-border-color: rgba(0, 0, 0, .2);
```

Comment:
There is no "global" variable available for a default border color, all other components define their own color -> picking the border color from another component seems wrong.